packages/shared/src/components/plus/PlusDesktop.tsx

```diff
@@ -85,11 +91,18 @@ export const PlusDesktop = ({
               element: 'h-[35rem]',
             }}
           />
-          <div className="flex justify-center">
-            <PlusTrustRefund />
-          </div>
+          {!isFreeTrialExperiment && !isPricesPending && (
+            <div className="flex justify-center">
+              <PlusTrustRefund />
+            </div>
+          )}
         </div>
       </div>
+      {isFreeTrialExperiment && !isPricesPending && !giftToUser && (
+        <div className="mx-auto mt-10 flex w-[62.5rem] gap-3">
```

Comment:
Would this work on smaller desktop screen sizes? At the point before it becomes tablet mode.

Reply:
Yes! ✔️ 

<img width="1031" alt="image" src="https://github.com/user-attachments/assets/90df40d1-7f31-4df8-9fec-832ef60f6a04" />